<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TypeFlashCardWidget</class>
 <widget class="QWidget" name="TypeFlashCardWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Review</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(30, 30, 30); color:white;</string>
  </property>
  <widget class="QPushButton" name="PassButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>570</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(20,80,20); border-radius:5px; border: 2px solid rgb(100,255,200);</string>
   </property>
   <property name="text">
    <string>Pass</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FailButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>570</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(80,20,20); border-radius:5px; border: 2px solid rgb(255,200,100)</string>
   </property>
   <property name="text">
    <string>Fail</string>
   </property>
  </widget>
  <widget class="QLabel" name="AnswerLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>360</y>
     <width>341</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(5,10,40); border: 2px solid rgb(30, 60, 120); border-radius:10px;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <property name="text">
    <string>Answer</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>480</width>
     <height>10</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>480</width>
     <height>0</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="QuestionLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>341</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(5,10,40); border: 2px solid rgb(30, 60, 120); border-radius:10px;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <property name="text">
    <string>Question</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>239</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:white;
 border-radius: 5px;
 border: 1px solid white;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FlipButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>570</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(20,20,80); border-radius:5px; border: 2px solid rgb(100,200,255);</string>
   </property>
   <property name="text">
    <string>Flip</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
